text/2019-11-13-follower-replication.md

```diff
@@ -0,0 +1,107 @@
+# Follower Replication
+
+## Summary
+This RFC introduces a new mechanism in Raft Protocol which allows a follower to send raft logs to other followers and learners.  The target of this feature is to reduce network transmission costs between different data centers in Log Replication.
```

Comment:
I think this can also reduce the pressure of the leader when it has many followers or learners.